modules/gapi/include/opencv2/gapi/media.hpp

```diff
@@ -15,6 +15,16 @@
 #include <opencv2/gapi/gframe.hpp>
 #include <opencv2/gapi/util/any.hpp>
 
+// Forward declaration
+namespace cv {
+namespace gapi {
+namespace s11n {
+    struct IOStream;
```

Comment:
We usually don't use indentations inside namespaces, right ?

Reply:
Done